### ISSUE TITLE
wrong example in chapter 'Creating a new plug-in extension point'

### ISSUE THREAD

**Issue by renemjrt** [dita-ot/docs#134]
At the end of chapter 'Creating a new plug-in extension point' there is an Example  of  how to create  plugin extension point. I have found it in docu for DITA OT 2.3 and 2.4 (http://www.dita-ot.org/2.4/dev_ref/plugin-newextensions.html)
Actual example:
```xml
<project xmlns:dita="http://dita-ot.sourceforge.net">
  <target name="dita2custom"
          depends="dita2custom.init,
                   {com.example.new-extensions.pre},
                   dita2xhtml"
          dita:extension="depends org.dita.dost.platform.InsertDependsAction">
    <dita:extension id="com.example.new-extensions.content"
                    behavior="org.dita.dost.platform.InsertAction"/>
  <target>
</project>
'''

but it is wrong, because during tranformation ANT will give an error. 
It should be corrected to :

```xml
<project` xmlns:dita="http://dita-ot.sourceforge.net">
  <target name="dita2custom"
           dita:depends="dita2custom.init,
                   {com.example.new-extensions.pre},
                   dita2xhtml"
          dita:extension="depends org.dita.dost.platform.InsertDependsAction">
    <dita:extension id="com.example.new-extensions.content"
                    behavior="org.dita.dost.platform.InsertAction"/>
  <target>
</project>
```

**Comment by georgebina** [comment]
In case it helps....
If you look at the current (dev) documentation
http://www.dita-ot.org/dev/dev_ref/plugin-newextensions.html
there is an "Edit this page" link at the end of the page, that will get you to [edit that topic online](https://www.oxygenxml.com/webapp-demo-aws/app/oxygen.html?url=github%3A%2F%2FgetFileContent%2Fdita-ot%2Fdocs%2Fdevelop%2Fdev_ref%2Fplugin-newextensions.dita&ditamap=github%3A%2F%2FgetFileContent%2Fdita-ot%2Fdocs%2Fdevelop%2Fuserguide-book.ditamap) and you can just make the suggested change and when you try to commit you will be asked to send a pull request with your suggestions.

Regards,
George

**Comment by renemjrt** [comment]
Hi George, 
thank you again for HINT to make a modification.  I nearly forgot it as you show this in DITA OT Days  :+1: 

Regards Rene

**Comment by infotexture** [comment]
Resolved w/ c4ad956 in #135.
